### ISSUE TITLE
Count function not working

### ISSUE THREAD

**Issue by ecstaticnate** [cayleygraph/cayley#632]
**Description**

Count function does not work.

**Steps to reproduce the issue:**

1. `cayley http --dbpath=./data/testdata.nq`
2. Navigate to web interface
3. Query `g.V("<fred>").Both("<follows>").Count()`

**Received results:**

```
{
 "result": null
}
```

**Expected results:**

```
{
 "result": 3
}
```

**Output of `cayley version` or commit hash:**

```
Cayley 0.6.1 built
```

**Environment details:**

Backend database: `LevelDB`

**Other notes:**

Querying `g.V("<fred>").Both("<follows>").All()` gives:

```
{
 "result": [
  {
   "id": "\u003cbob\u003e"
  },
  {
   "id": "\u003cemily\u003e"
  },
  {
   "id": "\u003cgreg\u003e"
  }
 ]
}
```

**Comment by ecstaticnate** [comment]
Closing because I realized that `Count()` must be operated on by `All()`. 

`g.V("<fred>").Both("<follows>").Count().All()`

Yields: 

```
{
 "result": [
  {
   "id": 1
  }
 ]
}
```

**Comment by dennwc** [comment]
It's fixed in 0.7, by the way. Still not released, though.
